invenio_communities/webpack.py

```diff
@@ -44,14 +44,13 @@
                 "invenio-communities-admin-featured": "./js/invenio_communities/administration/details.js",
             },
             dependencies={
-                "@ckeditor/ckeditor5-build-classic": "^16.0.0",
-                "@ckeditor/ckeditor5-react": "^2.1.0",
                 "@semantic-ui-react/css-patch": "^1.0.0",
                 "react-router-dom": "^6.3.0",
                 "react-invenio-forms": "^2.0.0",
                 "axios": "^0.21.0",
                 "formik": "^2.1.0",
                 "i18next": "^20.3.0",
+                "@tinymce/tinymce-react": "^4.3.0",
```

Comment:
minor: normally we shouldn't need this in the dependencies anymore, since we import the editor from react-invenio-forms, but to be checked